### PR TITLE
Printing hardhat config path in verbose mode

### PR DESCRIPTION
**Pull request by Bidon15**

### CODE REVIEW

File: packages/hardhat-core/src/internal/core/project-structure.ts [hunk 1]
```diff
@@ -9,11 +10,17 @@ import { ERRORS } from "./errors-list";
...
 
+const log = debug("hardhat:core:prj-structure");
+
 export function isCwdInsideProject() {
-  return (
-    findUp.sync(TS_CONFIG_FILENAME) !== null ||
-    findUp.sync(JS_CONFIG_FILENAME) !== null
-  );
+  const jsConfigPath = findUp.sync(JS_CONFIG_FILENAME);
+  const tsConfigPath = findUp.sync(TS_CONFIG_FILENAME);
```
Comment (Bidon15) [inline packages/hardhat-core/src/internal/core/project-structure.ts:17]: Maybe a good idea to relocate *ConfigPath vars out of functions as getUserConfigPath() contains the same vars.

File: packages/hardhat-core/src/internal/core/project-structure.ts [hunk 2]
```diff
@@ -9,11 +10,17 @@ import { ERRORS } from "./errors-list";
...
+  const path = jsConfigPath !== null ? jsConfigPath : tsConfigPath;
+  if (path !== null) {
```
Comment (Bidon15) [inline packages/hardhat-core/src/internal/core/project-structure.ts:19]: I didn't want to display the null path during project creation

**Comment by alcuadrado** [comment]
Hey, thanks for submitting this issue, @Bidon15! 

I wonder if this isn't a better place to log it: https://github.com/nomiclabs/hardhat/blob/d49afce4178f846241381168f2404b21e148ced3/packages/hardhat-core/src/internal/core/config/config-loading.ts#L44

**Comment by Bidon15** [comment]
> Hey, thanks for submitting this issue, @Bidon15!
> 
> I wonder if this isn't a better place to log it:
> 
> https://github.com/nomiclabs/hardhat/blob/d49afce4178f846241381168f2404b21e148ced3/packages/hardhat-core/src/internal/core/config/config-loading.ts#L44

Thanks for the guide @alcuadrado !
Yes. This look much better then my initial one.

File: packages/hardhat-core/src/internal/core/config/config-loading.ts [hunk 3]
```diff
@@ -41,7 +44,7 @@ export function loadConfigAndTasks(
     hardhatArguments !== undefined ? hardhatArguments.config : undefined;
 
   configPath = resolveConfigPath(configPath);
-
+  log(`Hardhat config path located at ${configPath}`);
```
Comment (alcuadrado) [inline packages/hardhat-core/src/internal/core/config/config-loading.ts:47]:
```suggestion
  log(`Loading Hardhat config from ${configPath}`);
```

What about this message instead? It makes it clear what was happening at the time the log was printed.

Comment (Bidon15) [reply packages/hardhat-core/src/internal/core/config/config-loading.ts]: Yeap. Clearer

**Comment by alcuadrado** [comment]
Oh, the linter is failing

**Pull request merged**